src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj

```diff
@@ -4,6 +4,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <PackageId>Microsoft.DotNet.Web.ProjectTemplates.$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)</PackageId>
```

Comment:
Shouldn't this be `Microsoft.DotNet.Web.ProjectTemplates.Spa.<version>`?

Reply:
I'm just doing what we did in [Microsoft.DotNet.Web.ProjectTemplates](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj#L5). The idea seems to be that the version we apply is "3.0" rather than "3.0.0-preview7".

Reply:
Right, but "Spa" is missing here in the package ID.

Reply:
Ah, I missed the relevant change.